### PR TITLE
PRM-71 threshold

### PR DESCRIPTION
**Pull request by AndrewRButcher**

### CODE REVIEW

**Review by LewisMitchell — APPROVED** [review]

File: app/connectors/PenaltiesConnector.scala [hunk 1]
```diff
@@ -30,7 +31,7 @@ class PenaltiesConnector @Inject()(httpClient: HttpClient,
   private def getPenaltiesDataUrl(enrolmentKey: String): String = s"/etmp/penalties/$enrolmentKey"
 
   def getPenaltiesData(enrolmentKey: String)(implicit hc: HeaderCarrier): Future[ETMPPayload] = {
-    println("[PenaltiesConnector][getPenaltiesData]: This connector is hit")
+    logger.debug("[PenaltiesConnector][getPenaltiesData]: This connector is hit")
```
Comment (LewisMitchell) [inline app/connectors/PenaltiesConnector.scala:34]: Do we still need this? Is it useful?

**Review by LewisMitchell — CHANGES_REQUESTED** [review]
LGTM. Perhaps a removal of a redundant log message.

File: app/connectors/PenaltiesConnector.scala (hunk 1 above)
Comment (AndrewRButcher) [reply app/connectors/PenaltiesConnector.scala]: Probably not, removed

**Review by LewisMitchell — APPROVED** [review]
Happy, merging.

**Pull request merged**